README.md

```diff
@@ -132,7 +132,32 @@ TodoItem.acts_as_list_no_update do
 end
 ```
 In an `acts_as_list_no_update` block, all callbacks are disabled, and positions are not updated. New records will be created with
- the default value from the database. It is your responsibility to correctly manage `positions` values. 
+ the default value from the database. It is your responsibility to correctly manage `positions` values.
+
+Also you can pass an argument as array of extracted calsses to disable from database updates. It might be any ActiveRecord class that is able to acts as list.
+```ruby
+class TodoList < ActiveRecord::Base
+  has_many :todo_items, -> { order(position: :asc) }
+end
+
+class TodoItem < ActiveRecord::Base
+  belongs_to :todo_list
+  has_many :todo_attachments, -> { order(position: :asc) }
+
+  acts_as_list scope: :todo_list
+end
+
+class TodoAttachment < ActiveRecord::Base
+  belongs_to :todo_list
+  acts_as_list scope: :todo_item
+end
+
+TodoItem.acts_as_list_no_update([TodoAttachment]) do
+  TodoItem.find(10).update(position: 2)
```

Comment:
Perhaps your example could show an update to `TodoList` also, with a comment next to it stating that callbacks will be called on that object.